Python/pythonrun.c

```diff
@@ -1484,6 +1484,69 @@ PyOS_CheckStack(void)
 
 #endif /* WIN32 && _MSC_VER */
 
+#if defined(__APPLE__)
+
+#define DEFAULT_PAGE_SIZE 2048
```

Comment:
What's DEFAULT_PAGE_SIZE? Is this the default number of pages in the stack of the main thread. 

If so, this value is wrong for framework builds, those use a stack of 0x1000000 bytes (8 MB, 4096 pages).

Reply:
@ronaldoussoren
Okay then let's return 8 MB stack size for this case.

Reply:
ref: https://cs.chromium.org/chromium/src/third_party/blink/renderer/platform/wtf/stack_util.cc?q=pthread_get_stacksize_np&sq=package:chromium&g=0&l=86